base/doc/ltnews41.tex

```diff
@@ -229,11 +229,28 @@ \subsection{Logging text command and symbol declarations}
 
 \section{Bug fixes}
 
-%\subsection{A fix}
-
-%% Some text
+\subsection{Fix the use of  \texttt{localmathalpabets}}
+
+In 2021 we introduced a method to overcome the problem that classic
+\TeX{} engines (but not the Unicode engines) have only a limited
+number of math alphabets available that got easily fill up simply by
+loading math font packages, even if their symbols got used only
+occasionally. The idea was to avoid allocating all math alphabets
+globally, but instead allow a number of them (defined by counter
+\texttt{localmathalpabets}) to vary from one formula to the next. This
+way different formulas can make use of different alphabets and chances
+are much higher that the processing a complex the document succeeds.
+See~\cite{41:ltnews34} for details.
+
+Unfortunately, the approach taken failed in some cases of nested
+formulas with the result that the wrong symbol glyphs were used
+(rather than, as before, your \LaTeX{} run stopped with a fatal error
```

Comment:
In some cases the original allocation would have worked (and the issue could be avoided by setting localmathalphabets to 0 if you get close enough to trigger the local mathalphabet mechanism without going over 16 fam . You could perhaps simply delete the parenthetic remark..